h5c/vic/src/vic-webapp/src/app/container-view/container-view.template.html

```diff
@@ -6,18 +6,29 @@
           <clr-alert-item *ngIf="errorObj.type === 'ssl_cert'">
             <span class="alert-text">
               <!-- TODO: i18n -->
-              Failed to verify the VIC machine server at endpoint https://{{ errorObj.payload }}:8443. Your browser may be blocking the connection because of the untrusted SSL certificate.
+              Failed to verify the VIC machine server at endpoint https://{{ errorObj.payload }}:8443. Please check the following:<br/>
+              1. VIC appliance VM is up, but unreachable due to network misconfigurations<br/>
+              2. VIC appliance VM is up and reachable, but API endpoint is unreachable due to possible issues that include, but not limited to:<br/>
+              &nbsp;&nbsp;- Network misconfiguration<br/>
+              &nbsp;&nbsp;- Firewall rules<br/>
+              &nbsp;&nbsp;- Endpoint being down<br/>
+              3. Your browser may be blocking the connection because of the untrusted SSL certificate. Click <a href="https://{{ errorObj.payload }}:8443/container/hello" target="_blank">here</a> to fix it<br/>
```

Comment:
How about something more like:

```html
Your browser may be blocking the connection because of an untrusted SSL
certificate. <a href="https://{{ errorObj.payload }}:8443/container/hello"
target="_blank">View the API directly in your browser</a> to see if any
warnings are presented.
```

Reply:
"Endpoint being down" ==> "API endpoint is down"